tests/rules/test_new_lines.py

```diff
@@ -58,3 +60,28 @@ def test_dos_type(self):
         self.check('\r\n', conf)
         self.check('---\ntext\n', conf, problem=(1, 4))
         self.check('---\r\ntext\r\n', conf)
+
+    def test_auto_type(self):
+        conf = ('new-line-at-end-of-file: disable\n'
+                'new-lines: {type: auto}\n')
+
+        self.check('', conf)
```

Comment:
Why this?

Reply:
It's true that this check *may* have sense in the case of the `new-lines` rule, because `''` is a kind of line. OK to keep it!